The tables have been fixed and set phone and address field in the reg…

IvailoA05 opened the pull request:
…istration screen

Commit:
The tables have been fixed and set phone and address field in the registration screen

aruseva commented:
1. In User.cs,  change validation for Address and PhoneNumber
2. In frmAdminPanel  - add Address and PhoneNumber

TheSovietBear committed:
We added address and phone to this bitch

Dumbass fucker didnt want to push from visual studio 2022 so we decided to commit from git tiself like total monkeys.

TheSovietBear committed:
Removed something. Idk what tho

IvailoA05 committed:
Added brackets to avoid errors in the isAdmin statement
